src/materials/GenericMoltresMaterial.C

```diff
@@ -162,13 +166,25 @@ GenericMoltresMaterial::dummyConstruct(std::string & property_tables_root,
   {
     std::vector<Real> temperature;
     std::string file_name = property_tables_root + _file_map[xsec_names[j]] + ".txt";
+    
     const std::string & file_name_ref = file_name;
     std::ifstream myfile(file_name_ref.c_str());
-
     auto o = _vec_lengths[xsec_names[j]];
 
     _xsec_map[xsec_names[j]].resize(o);
     _xsec_spline_interpolators[xsec_names[j]].resize(o);
+
+    //chi_d backwards compatibility on unit tests:
+    if (xsec_names[j]=="CHI_D" and not myfile.good())
+    {
+      for (decltype(o) k = 0; k < o; ++k)
+        _xsec_map["CHI_D"][k] = (Real)0.0L;
```

Comment:
We don't want to specify a `long double` here. Additionally, there's no need to explicitly cast to Real (double). This is done implicitly. Just set equal to `0.` or `0.0`.